cedar-policy-generators/src/abac.rs

```diff
@@ -296,6 +296,84 @@ impl ConstantPool {
         .map(SmolStr::new)
     }
 
+    // Generate a roughly valid datetime string
+    fn arbitrary_datetime_str_inner(&self, u: &mut Unstructured<'_>) -> Result<String> {
+        let mut result = String::new();
+        // Generate YYYY-MM-DD
+        let y = u.int_in_range(0..=999)?;
```

Comment:
this doesn't include the current year, should this be `0..=3000` or something?

Reply:
Good catch. Fixed.